test/SlowTests/Voron/Issues/RDBCL_1977.cs

```diff
@@ -4,93 +4,95 @@
 using Xunit;
 using Xunit.Abstractions;
 
-namespace SlowTests.Voron.Issues;
-
-public class RDBCL_1977 : StorageTest
+namespace SlowTests.Voron.Issues
```

Comment:
Please revert the changes made here. I already did it on 4.2 branch so it will be merged to other branches

https://github.com/ravendb/ravendb/pull/13086

Reply:
Note: our CI environments already have .net sdk 6.0 installed so it will compile correctly

Reply:
done